### ISSUE TITLE
response content types - all request seem to return `text/html`

### ISSUE THREAD

**Issue by RoboSparrow** [gardners/surveysystem#159]
all request seem to return `text/html`

example headers for an expected json response
```bash
 curl -I http://localhost:3099/surveyapi/nextquestion?sessionid=56e86bec-0000-0000-0fc3-0da3df13f712
HTTP/1.1 200 OK
Access-Control-Allow-Origin: *
Access-Control-Allow-Methods: HEAD, GET, OPTIONS
Access-Control-Expose-Headers: Content-Range, Date, Etag, Cache-Control, Last-Modified
Access-Control-Allow-Headers: Content-Type, Origin, Accept, Range, Cache-Control
Access-Control-Max-Age: 600
Timing-Allow-Origin: *
Content-Type: text/html # <===== application/json
Content-Length: 128
Date: Mon, 29 Apr 2019 07:03:08 GMT
Server: lighttpd/1.4.45
```

it should be: 

```c
static const char *const pages[PAGE__MAX] = {
  "newsession", // text/plain
  "addanswer",  // application/json
  "updateanswer", // application/json
  "nextquestion", // application/json
  "delanswer", // text/plain
  "delsession",// text/plain
  "accesstest",// text/plain
  "fastcgitest", // text/plain
  "analyse"  // application/json
};
```

**Comment by RoboSparrow** [comment]
related to #187 ,  all responses should be `text/plain`

**Comment by RoboSparrow** [comment]
closing merged into #159
